### PR TITLE
Elasticsearch: fix typo in signup url

### PR DESCRIPTION
**Pull request by saarikabhasi**
## PR Summary

This PR fixes Elastic Cloud sign up url in Elasticsearch Vector Database example [notebooks](https://github.com/openai/openai-cookbook/tree/main/examples/vector_databases/elasticsearch)

### CODE REVIEW

**Review by shyamal-anadkat — APPROVED** [review]
lgtm, thanks!

File: examples/vector_databases/elasticsearch/elasticsearch-retrieval-augmented-generation.ipynb [hunk 1]
```diff
@@ -879,7 +879,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.12"
+   "version": "3.11.4"
```
Comment (simonpfish) [inline examples/vector_databases/elasticsearch/elasticsearch-retrieval-augmented-generation.ipynb:882]: is this change intentional?

File: examples/vector_databases/elasticsearch/elasticsearch-semantic-search.ipynb [hunk 2]
```diff
@@ -803,7 +803,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.12"
+   "version": "3.11.4"
```
Comment (simonpfish) [inline examples/vector_databases/elasticsearch/elasticsearch-semantic-search.ipynb:806]: likewise here

**Review by simonpfish — CHANGES_REQUESTED** [review]
Are these python version changes intentional?

File: examples/vector_databases/elasticsearch/elasticsearch-retrieval-augmented-generation.ipynb (hunk 1 above)
Comment (simonpfish) [inline examples/vector_databases/elasticsearch/elasticsearch-retrieval-augmented-generation.ipynb:882]:
```suggestion
   "version": "3.10.12"
```

File: examples/vector_databases/elasticsearch/elasticsearch-semantic-search.ipynb (hunk 2 above)
Comment (simonpfish) [inline examples/vector_databases/elasticsearch/elasticsearch-semantic-search.ipynb:806]:
```suggestion
   "version": "3.10.12"
```

**Review by simonpfish — APPROVED** [review]

**Comment by joemcelroy** [comment]
thank you!

**Pull request merged**